armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -166,6 +164,15 @@ def outage(self, factor=1.0):
         self.moved = []
         return moved
 
+    def _tryCallAssemblyRotation(self, rotationMethod):
+        """Determine the best way to call the rotation algorithm.
+
+        Some callers take no arguments, some take the fuel handler.
+        """
+        sig = inspect.signature(rotationMethod)
+        args = (self,) if sig.parameters else ()
+        rotationMethod(*args)
```

Comment:
So, the best solution here would be to unify all of the `rotationMethod`s so they all take one parameter, even if they don't use it.

That would be the cleanest form of the code.

Thoughts?

Reply:
Will do!

Reply:
Oh, okay, cool. That wasn't a demand. But, awesome.